cifar/main.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import resnet
+import numpy as np
```

Comment:
You could probably remove numpy from this example to simplify things. Looks like it is barely used and easily replaceable with mlx